honssh/base_auth_handler.py

```diff
@@ -35,29 +35,33 @@
 
 class Base_Auth():
     
-    def __init__(self, server):
+    def __init__(self, server, name):
         self.server = server
+        self.name = name
         self.auth_plugin = None
         self.cfg = self.server.cfg
-        
+
+        self.connection_timeout = 10
```

Comment:
Doesn't this need to be the connection_timeout from the config file? Rather than fixed?
Or am I missing something?